libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -83,8 +83,7 @@ AP_Arming::AP_Arming(const AP_AHRS &ahrs_ref, Compass &compass,
     ahrs(ahrs_ref),
     _compass(compass),
     _battery(battery),
-    armed(false),
-    arming_method(NONE)
+    armed(false)
```

Comment:
Unneeded initializer

Reply:
Yes, I saw this pointless initialiser.  I thought, "I'll remove that when I change this to use singletons".

Reply:
My thought was: you are already modifying the line, no reason to not just delete it :)